baricadr/model/repos.py

```diff
@@ -104,8 +113,10 @@ def freeze(self, path, force=False, dry_run=False):
         # TODO [HI] test force mode in freeze
 
         current_app.logger.info("Asked to freeze '%s'" % path)
+        if not (force or self.can_freeze):
```

Comment:
the condition looks strange, are you sure?

Reply:
Well, this stop the freeze if we are not forcing or it the repo is not freezable. Maybe we should remove the force check? Make it impossible to freeze non-freezable repos?

Reply:
ah yes ok, it makes sense now